data-gathering/bot.py

```diff
@@ -6,72 +6,56 @@
 
 import urllib
 import sys
-import calendar
 import json
 import datetime
 from urllib.request import urlopen
 import urllib.parse
-import string
-import json
 import chardet
 from datetime import datetime
-from query import store_articles, fetch_wikidata_info
+from query import store_articles, fetch_wikidata_info, get_id_and_subjects_and_grade
 
 # Open and read the JSON configuration file and access the configuration data as a dictionary
 with open("wikipedia_config.json", "r") as config_file:
    wikipedia_config = json.load(config_file)
 
 
 # Accept 
-if len(sys.argv) > 1:
-    WIKIPEDIA_LANGUAGE  = sys.argv[1]
+if len(sys.argv) > 2:
+   WIKIPEDIA_LANGUAGE  = sys.argv[1]
+   CODE = sys.argv[2]
```

Comment:
we could provide a more descriptive name for this variable like COUNTRY or WIKIDATA_COUNTRY_ID

Reply:
Noted